CHANGELOG.asciidoc

```diff
@@ -87,6 +69,47 @@ https://github.com/elastic/beats/compare/v5.0.0...5.0[Check the HEAD diff]
 
 ////////////////////////////////////////////////////////////
 
+[[release-notes-5.0.1]]
+=== Beats version 5.0.1
+https://github.com/elastic/beats/compare/v5.0.0...v5.0.1[View commits]
+
+The list below covers the changes between 5.0.0 GA and 5.0.1 only.
+
+==== Bugfixes
+
+*Metricbeat*
+
+- Fix `system.process.start_time` on Windows. {pull}2848[2848]
+- Fix `system.process.ppid` on Windows. {issue}2860[2860]
+- Fix system process metricset for Windows XP and 2003. `cmdline` will be unavailable. {issue}1704[1704]
+- Fix access denied issues in system process metricset by enabling SeDebugPrivilege on Windows. {issue}1897[1897]
+- Fix system diskio metricset for Windows XP and 2003. {issue}2885[2885]
+
+*Packetbeat*
+
+- Fix 'index out of bounds' bug in Packetbeat DNS protocol plugin. {issue}2872[2872]
+
+*Filebeat*
+
+- Fix registry cleanup issue when files falling under ignore_older after restart. {issue}2818[2818]
+
+
+==== Added
+
+*Metricbeat*
+
+- Add username and password config options to the PostgreSQL module. {pull}2889[2890]
+- Add username and password config options to the MongoDB module. {pull}2889[2889]
+- Add system core metricset for Windows. {pull}2883}[2883]
+
+*Packetbeat*
```

Comment:
I think you need an empty line after the title, otherwise it will look broken in asciidoc.